game/hud/src/services/actions/compass.ts

```diff
@@ -33,6 +33,32 @@ export function getCompassData() {
   };
 }
 
+export function getCompassFacingData() {
+  let facing: number = client.facing % 360;
+  if (facing < 0) {
```

Comment:
facing can't be < 0 here. Looking at the line above, the sign of the result of the remainder operation is the same as the dividend.

Reply:
the `client.facing` value is sometimes negative e.g. `-60`... running `-60 % 360` would give `-60`, the facing is then converted to a positive angle  `-60` is same as `300` .

Reply:
yea, you're right... idk why I was thinking about that backwards